parser/ast/ddl.go

```diff
@@ -73,14 +73,20 @@ const (
 	DatabaseOptionCharset
 	DatabaseOptionCollate
 	DatabaseOptionEncryption
+	DatabaseSetTiFlashReplica
 	DatabaseOptionPlacementPolicy = DatabaseOptionType(PlacementOptionPolicy)
 )
 
 // DatabaseOption represents database option.
 type DatabaseOption struct {
-	Tp        DatabaseOptionType
-	Value     string
-	UintValue uint64
+	Tp             DatabaseOptionType
+	Value          string
+	UintValue      uint64
+	TiFlashReplica *TiFlashReplicaSpec
+}
+
+func restoreTiFlashStmt(ctx *format.RestoreCtx, tiFlashReplica *TiFlashReplicaSpec) {
```

Comment:
What is this line?

Reply:
Some later work...I should remove them before have this PR...